pkg/adapter/containers.go

```diff
@@ -834,3 +834,30 @@ func (r *LocalRuntime) Prune(ctx context.Context, maxWorkers int, force bool) ([
 	}
 	return pool.Run()
 }
+
+// Port displays port information about existing containers
+func (r *LocalRuntime) Port(c *cliconfig.PortValues) ([]*Container, error) {
```

Comment:
Why is this returning []*Container?

Reply:
It seems like this ought to return a map of container ID to port mappings the container has